dontforgit/dontforgit-config
# Feat/automatic updates to npm repositories

## Changelog
- Add the scripts for automatically updating NPM dependencies
  - `autoup_minor` - Updates all dependencies to the latest minor version or patch version.
  - `autoup` - Updates all dependencies to the latest version.

--- 

## How to Test
1. Add various outdated packages to your `package.json` file. 
2. Change your `engines.node` to be a newer version of Node
3. Run: `autoup_minor`

- [x] Verify that some packages have been updated, and that those packages only applied minor or patch releases. 

4. Run: `autoup`

- [x] Verify that some packages have been updated to their latest major release.
